src/GPU/CalculateEnergyCUDAKernel.cu

```diff
@@ -278,7 +278,7 @@ __global__ void BoxInterGPU(int *gpu_cellStartIndex,
         if(electrostatic) {
           double qi_qj_fact = gpu_particleCharge[currentParticle] * gpu_particleCharge[neighborParticle];
           if(qi_qj_fact != 0.0) {
-            static const double qqFact = 167000.0;
+            static const double qqFact = 167103.208067979;
```

Comment:
We should define the constant `qqFact` in one place, to avoid problem in later developments. Our approach to define/pass `qqFact` is not consistence. For example in `CallBoxForceReciprocalGPU` and some other GPU kernal calling, we are passing `num::qqFact` but other places like here, we define them internally.

Reply:
I will revise the code to pass this as a parameter to both functions and resubmit the patch. Thanks.